### PR TITLE
Add .env for specifying aws logging specifics

### PR DESCRIPTION
**Pull request by jayjb**
## Proposed changes

This commit allows a `.env` file that is used to define some of the AWS logging specifics. We do have defaults so it is not required.

## Types of changes

What types of changes does your code introduce to this repository?
_Put an `x` in the boxes that apply_

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update ([here](https://github.com/thinkst/canarytokens-docker/discussions/165))

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes (if applicable)
- [x] I have run pre-commit (`pre-commit` in the repo)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ Thanks, @mamisano for catching a silly issue using the above 🙏
 
 We are going to track some new features/additions here so that it is quick and easy to see what has been recently added.
 
-- we have moved the "What's new?" to Github Discussions. Go check it out over ()[]
+- we have moved the "What's new?" to Github Discussions. Go check it.
```
Comment (wleightond) [inline README.md:205]: We should either remove that sentence or fill in the link.

```suggestion
- we have moved the "What's new?" to Github Discussions.
```

```suggestion
- we have moved the "What's new?" to Github Discussions. Check it out (here)[https://github.com/thinkst/canarytokens-docker/discussions].
```

File: docker-compose-aws-logging-letsencrypt.yml [hunk 2]
```diff
@@ -7,8 +7,8 @@ services:
     logging:
       driver: awslogs
       options:
-        awslogs-region: eu-west-1
-        awslogs-group: canarytokens-dev
+        awslogs-region: ${CANARYTOKENS_LOGGING_REGION:-eu-west-1}
```
Comment (wleightond) [inline docker-compose-aws-logging-letsencrypt.yml:10]: My first thought was that we may want to use the standard `CANARY_` prefix, but as this is not being included in the settings objects I suspect it's better as it is.

**Review by wleightond — APPROVED** [review]
I've suggested one small `README.md` fix

Approving to unblock you

File: docker-compose-aws-logging-letsencrypt.yml (hunk 2 above)
Comment (jayjb) [reply docker-compose-aws-logging-letsencrypt.yml:10]: hmmm its a good thought to raise. I hadn't considered it. I'll leave for now.

**Pull request merged**